Libraries/ReactNative/I18nManager.js

```diff
@@ -10,13 +10,11 @@
 
 import NativeI18nManager from './NativeI18nManager';
 
-const i18nConstants: {|
+function getI18nManagerConstants(): {|
```

Comment:
?

Reply:
git makes it look weird, if you look at the file instead of changes you will understand why
I shouldn't have change `i18nConstants` to something else probably
I can revert the name change if needed

Reply:
also I just noticed now the code being reviewed here is before [this commit](https://github.com/facebook/react-native/pull/33956/commits/4406ae3e0161881b51bd4730bee57298917c2edc) is applied